engine/src/flutter/lib/web_ui/dev/steps/copy_artifacts_step.dart

```diff
@@ -57,7 +57,7 @@ class CopyArtifactsStep implements PipelineStep {
     };
     final Uri url = Uri.https(
       'storage.googleapis.com',
-      '${realmComponent}flutter_infra_release/flutter/$gitRevision/flutter-web-sdk.zip',
+      '${realmComponent}flutter_infra_release/flutter/${isPostsubmit ? contentHash : gitRevision}/flutter-web-sdk.zip',
```

Comment:
You might be able to just do `realm == LuciRealm.Try ? gitRevision : contentHash` instead of reading the `LUCI_PR` environment variable. It's probably equivalent functionally, so it's up to you.

Reply:
Ah that's better.

I made it a negative so that, if you use this tool locally, it defaults to git revision instead of content hash.

Reply:
The way I'm reading the code, only `LuciRealm.Try` will use `gitRevision`. Does local development use `LuciRealm.Try` or `LuciRealm.Unknown`?

Reply:
This code path is never going to be exercised locally (and your actual local build will not ever be in GCS) so that point is moot.

I think the staging realm is postsubmit + `bringup: true` so I think that should be contentHash as well. Hence, only the try realm should actually use `gitRevision` right now, IMO

Reply:
Ok that makes sense.

My comment was a response to:

> I made it a negative so that, if you use this tool locally, it defaults to git revision instead of content hash.

So if that point is moot, please carry on 🙂